src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.product-card {
+  padding: 32px 16px 16px;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  transition: all 0.2s;
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The hover effect on the `.product-card` class uses `transform: scale(1.2)`, which might cause layout shifts and affect the surrounding elements. Consider using a smaller scale value or alternative effects to minimize layout disruptions.